index.js

```diff
@@ -298,16 +298,16 @@ function diffArguments(thisType, otherType, thisField, otherField, options) {
     let newArgsMap = getArgumentMap(otherField.args);
     oldArgsMap.forEach(function(value, key) {
         if (!newArgsMap.has(key)) {
-            const description = format('Argument `"{0} : {1}"` on type {2}.{3} has been removed from {4}.', value, key, thisType, thisField.name, options.labelForThis);
+            const description = format('Argument missing from {0} : `"{1} : {2}"` on field {3}.{4}.', options.labelForThis, key, value, thisType, thisField.name);
```

Comment:
Let's format it like this (fields substituted for readability):
Argument missing from existing schema: `Query.VideoByGenre(genre: String!)`.

That makes it a bit more succinct.